packaging/anaconda-webui.spec.in

```diff
@@ -11,7 +11,19 @@ BuildRequires:  libappstream-glib
 BuildRequires:  make
 BuildRequires:  gettext
 
-%global anacondacorever 40.20
+%if %{?fedora} < 40
+# Web UI is not supported on Fedora < 40
+%global anacondacorever 1000
```

Comment:
This is to require non-existing version, thus kill the install :thinking: I am not a packager so no idea if there is better way and quick read through guidelines did not yield any results so I guess all good?

Reply:
So often we build on some other Fedora the srpms, and not having the variable defined kills rpmbuild. I dont know what's the 'correct' approach here.

Reply:
It "kills" (i.e. fails) rpmbuild because it can't parse `Requires: anaconda-core  >= %{anacondacorever}` then? That's strange, it shouldn't care about this at srpm build time. But if/as it apparently does, a cleaner way would be either to conditionalize that Requires like on `if 0%{?anacondacorever}` (more correct and cleaner), or setting the default value to `0` (sloppy, but easier). I suppose you don't want to render this uninstallable on earlier Fedoras if you try and build them there?

Reply:
Yes, I would like to make this uninstallable for < 40. I would like to allow people to build it for >= 40, even though it's decided to not be the default installer for 40, still it makes sense to enable that for testing purposes.